prime-router/quick-test.sh

```diff
@@ -94,13 +94,15 @@ then
 fi
 
 # run arizona
+AZ_FILE_SEARCH_STR="/az.*\.csv"
+PIMA_FILE_SEARCH_STR="/pima.*\.csv"
 if [ $RUN_AZ -ne 0 ]
```

Comment:
Change to make sure we grab the .csv file to do the regression comparison, not the .hl7 file.